pkg/execution/history/history.go

```diff
@@ -100,3 +101,17 @@ type Result struct {
 	SizeBytes   int              `json:"response_size_bytes"`
 	Stack       []map[string]any `json:"stack"`
 }
+
+// HistorySkip represents a row in the workflow_run_skips table
+type HistorySkip struct {
+	AccountID     uuid.UUID
+	BatchID       *ulid.ULID
+	Cron          *string
+	EventID       ulid.ULID
+	OriginalRunID *ulid.ULID
+	RunID         *ulid.ULID
+	RunSkippedAt  time.Time
+	Status        string
```

Comment:
Maybe rename this to make it clear that's a reason and not a run status
```suggestion
	Reason        string
```